### DISCUSSION TITLE
Can we add dynamically new staticPasswords after the dex is started

### DISCUSSION THREAD

**Discussion by morarucostel** [dexidp/dex#2230] [Q&A]
We want to use the dex in a basic-auth style, with staticPasswords and we want to make allow the end-users to register new accounts and just inject the collected data (email, username and hashed pass) in dex - to allow he newly registered user to login.

How can we achieve this use-case? Does dex support adding dynamically new users using staticPasswords?

**Comment by nabokihms** [comment]
Yes, it is possible. You can do it via grpc API. There is amore about it here https://dexidp.io/docs/api/

**Reply by Daerendor** [reply]
Are there any plans to make a web interface for managing Dex staticPasswords entries?
If not, do you know a web client made for Dex?

**Reply by nabokihms** [reply]
There is neither official web client, nor the one I know about.
